web/app/src/views/Home.vue

```diff
@@ -17,13 +17,26 @@
               <Activity v-if="showAverageResponseTime" class="h-5 w-5" />
               <Timer v-else class="h-5 w-5" />
             </Button>
+
             <Button variant="ghost" size="icon" @click="refreshData" title="Refresh data">
               <RefreshCw class="h-5 w-5" />
             </Button>
+
+            <Button
+              variant="ghost"
+              size="icon"
+              @click="toggleDefaultCollapse"
```

Comment:
This is just for groups, right? Meaning it only applies if the user sorts by group.

Isn't this button kind of useless? I think I would rather have this as a configuration just in the configuration, and not allow people to toggle it in the UI